homeassistant/components/remote_rpi_gpio/light.py

```diff
@@ -0,0 +1,229 @@
+"""Support for LED lights that can be controlled using PWM."""
+from __future__ import annotations
+
+import logging
+
+from pwmled import Color
+from pwmled.driver.gpio import GpioDriver
+from pwmled.led import SimpleLed
+from pwmled.led.rgb import RgbLed
+from pwmled.led.rgbw import RgbwLed
+import voluptuous as vol
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_HS_COLOR,
+    ATTR_TRANSITION,
+    PLATFORM_SCHEMA,
+    ColorMode,
+    LightEntity,
+    LightEntityFeature,
+)
```

Comment:
So far as I known, integrations using GPIO are not allowed anymore; https://github.com/home-assistant/architecture/blob/master/adr/0019-GPIO.md

I think you should use ESP-Home or deploy this as an custom integration.

Reply:
As you can see in this [line of code](https://github.com/soldag/python-pwmled/blob/main/pwmled/driver/gpio.py#L23) the underlying light driver makes use of pigpio, just like [remote_rpi_gpio](https://github.com/home-assistant/core/blob/dev/homeassistant/components/remote_rpi_gpio/__init__.py#L3) code does. Connection is made with the local pigpio deamon, or can be configured to connect to a remote system, so the IO is not GPIO but via network socket. Because of this AD0019 does not apply on this code.